js/modules/require_impl.go

```diff
@@ -195,3 +197,11 @@ func getPreviousRequiringFile(vu VU) (string, error) {
 	}
 	return result, nil
 }
+
+// sets the provided promise in such way as to ignore falures
+// this is mostly needed as failures are handled separately and we do not want those to lead to stopping the event loop
+func promisesThenIgnore(rt *sobek.Runtime, promise *sobek.Promise) {
+	cal, _ := sobek.AssertFunction(rt.ToValue(promise).ToObject(rt).Get("then"))
+	handler := rt.ToValue(func(_ sobek.Value) {})
+	_, _ = cal(rt.ToValue(promise), handler, handler)
+}
```

Comment:
_Nit; I suspect that's what you really wanted to write 😅_ 

```suggestion
func promisesThenIgnore(rt *sobek.Runtime, promise *sobek.Promise) {
	call, _ := sobek.AssertFunction(rt.ToValue(promise).ToObject(rt).Get("then"))
	handler := rt.ToValue(func(_ sobek.Value) {})
	_, _ = call(rt.ToValue(promise), handler, handler)
}
```

Reply:
Is it fine to ignore the boolean return? I guess it's because we now it's a Promise, and `then` must always be a function, right? 🤔